### DISCUSSION TITLE
Is it possible to rename the configuration file?

### DISCUSSION THREAD

**Discussion by johnmapeson** [pqrs-org/Karabiner-Elements#4044] [Q&A] [answered]
Currently the configuration file is called `karabiner.json`:

```
$HOME/.config/karabiner/karabiner.json
```

Is it possible to rename it, so that it will like this:

```
$HOME/.config/karabiner/custom-name.json
```

?

**Comment by tekezo** [comment, marked as answer]
No, you cannot rename karabiner.json as its name is fixed.
